miqa/core/rest/scan_decision.py

```diff
@@ -50,6 +51,22 @@ def create(self, request, **kwargs):
 
         request_data['scan'] = scan
         request_data['creator'] = request.user
+        if (
+            'artifacts' in request_data
+            and 'present' in request_data['artifacts']
+            and 'absent' in request_data['artifacts']
+        ):
+            request_data['user_identified_artifacts'] = {
+                artifact_name: (
+                    1
+                    if artifact_name in request_data['artifacts']['present']
+                    else 0
+                    if artifact_name in request_data['artifacts']['absent']
+                    else -1
```

Comment:
These (-1, 0, 1) values clearly have some semantic meaning, but I have to assume it's 1=present, -1=absent, 0=not set. An enum or some constants would make reading this code easier, and using a string-based enum (`"PRESENT", "ABSENT", "UNDEFINED") would make reading the data/frontend code easier as well.

Reply:
Good idea!